src/main/kotlin/com/terraformation/backend/tracking/api/ObservationsController.kt

```diff
@@ -1473,11 +1475,12 @@ data class ScheduleObservationRequestPayload(
     val endDate: LocalDate,
     @Schema(description = "Which planting site this observation needs to be scheduled for.")
     val plantingSiteId: PlantingSiteId,
-    @Schema(
-        description =
-            "If this observation should only cover specific parts of the planting site, the IDs " +
-                "of the subzones it should include.")
-    val requestedSubzoneIds: Set<PlantingSubzoneId>? = null,
+    @field:NotEmpty
+    @ArraySchema(
+        arraySchema =
+            Schema(description = "The IDs of the subzones this observation should cover."),
+        minItems = 1)
+    val requestedSubzoneIds: Set<PlantingSubzoneId>,
```

Comment:
The web app already always populates this field, so there shouldn't be any changes required in terraware-web to account for this API schema change.